heartbeat/monitors/wrappers/wrappers.go

```diff
@@ -381,3 +377,47 @@ func makeAddSummary() jobs.JobWrapper {
 		}
 	}
 }
+
+type EventMatcher func(event *beat.Event) bool
+
+func addBrowserSummary(sf stdfields.StdMonitorFields, match EventMatcher) jobs.JobWrapper {
+	return func(job jobs.Job) jobs.Job {
+		return func(event *beat.Event) ([]jobs.Job, error) {
+			cont, jobErr := job(event)
+
+			if match != nil && !match(event) {
+				return cont, jobErr
+			}
+
+			status, err := event.GetValue("monitor.status")
+			if err != nil {
+				return nil, fmt.Errorf("could not wrap summary for '%s', no status assigned: %w", sf.ID, err)
+			}
+
+			up, down := 1, 0
+			if monitorstate.StateStatus(status.(string)) == monitorstate.StatusDown {
+				down, up = up, down
```

Comment:
```suggestion
				up, down = 0, 1
```

This is more clear, and in terms of performance, either negligible or zero due to compiler optimizations